frontend/package.json

```diff
@@ -7,33 +7,27 @@
     "test": "npm run typecheck && jest",
     "dev": "next",
     "dev:local-wamp": "env WAMP_NEAR_EXPLORER_URL=ws://localhost:8080/ws npm run dev",
-    "dev:local-wamp-with-legacy-sync": "env NEAR_EXPLORER_DATA_SOURCE=LEGACY_SYNC_BACKEND npm run dev:local-wamp",
     "dev:local-wamp-with-indexer": "env NEAR_EXPLORER_DATA_SOURCE=INDEXER_BACKEND npm run dev:local-wamp",
```

Comment:
We don't need `NEAR_EXPLORER_DATA_SOURCE` anymore, so we can merge and get rid of `-with-indexer` variants to simplify things. Indexer backend is the only source of data (analytics database is not an alternative source, it is just a second dependency)